generators/server/templates/src/test/java/package/gateway/responserewriting/SwaggerBasePathRewritingFilterTest.java.ejs

```diff
@@ -99,14 +99,14 @@ class SwaggerBasePathRewritingFilterTest {
         context.setResponseGZipped(true);
         context.setResponse(response);
 
-        context.setResponseDataStream(new ByteArrayInputStream(gzipData("{\"basePath\":\"/\"}")));
+        context.setResponseDataStream(new ByteArrayInputStream(gzipData("{\"servers\":[{\"url\":\"http://localhost:8080/services\"}]}")));
 
         filter.run();
 
         assertEquals("UTF-8", response.getCharacterEncoding());
 
         InputStream responseDataStream = new GZIPInputStream(context.getResponseDataStream());
         String responseBody = IOUtils.toString(responseDataStream, StandardCharsets.UTF_8);
-        assertEquals("{\"basePath\":\"/service1\"}", responseBody);
+        assertEquals("{\"servers\":[{\"url\":\"http://localhost:8080/services/service1\"}]}", responseBody);
     }
```

Comment:
add 2 more tests:

Without servers:

```
context.setResponseDataStream(new ByteArrayInputStream(gzipData("{}")));
```

Without URLs:

```
context.setResponseDataStream(new ByteArrayInputStream(gzipData("{\"servers\":[]}")));
```

Then, change your original code according to the tests

Reply:
@pascalgrimaud what should be the results in these two cases ?

Reply:
Maybe this `assertEquals("{\"servers\":[{\"url\":\"http://localhost:8080/services\"}]}", context.getResponseBody());
`

Reply:
the result should be inchanged:

```java
assertEquals("{\"servers\":[{\"url\":\"http://localhost:8080/services/service1\"}]}", responseBody);
```

Reply:
ok